frontend/rpm/template.go

```diff
@@ -326,6 +326,17 @@ func (w *specWrapper) Install() fmt.Stringer {
 		copyArtifact(`%{buildroot}/%{_mandir}`, p, cfg)
 	}
 
+	sort.Strings(w.Spec.Artifacts.ConfigDirectories)
+	for _, p := range w.Spec.Artifacts.ConfigDirectories {
+		dir := filepath.Join(`%{buildroot}/%{_sysconfdir}`, p)
+		fmt.Fprintln(b, "mkdir -p", dir)
```

Comment:
Need to make sure the dirs are quoted for the shell.

```suggestion
		fmt.Fprintf(b, "mkdir -p %q\n", dir)
```

Reply:
good catch thanks!